zipline/src/jvmMain/resources/META-INF/proguard/zipline.pro

```diff
@@ -0,0 +1,36 @@
+### ZIPLINE
+
+-keepclassmembers class app.cash.zipline.QuickJsException { * ; }
```

Comment:
This is insufficient as the type is looked up by FQCN: https://github.com/cashapp/zipline/blob/d7ae3d7bde4e1beee119a0d460cd22cc223f3b77/zipline/native/Context.cpp#L94-L95
```suggestion
-keep,allowoptimization class app.cash.zipline.QuickJsException { * ; }
```